webapp/src/components/file_preview/file_preview_header.tsx

```diff
@@ -0,0 +1,158 @@
+                        <div
+                            style={{
+                                display: 'flex',
+                                fontSize: 13,
+                                lineHeight: 1.38463,
+                                fontWeight: 400,
+                            }}
+                        >
+                            <span
+                                style={{
+                                    color: '#606060',
+                                    fontWeight: 700,
+                                    paddingRight: 4,
+                                    whiteSpace: 'nowrap',
+                                    overflow: 'hidden',
+                                    textOverflow: 'ellipsis',
+                                }}
+                            >
+                                {channelName}
+                            </span>
+                        </div>
+                    </div>
+                </div>
+                <div
+                    id='headerActions'
+                    style={{
+                        flex: '0 0 auto',
+                        display: 'flex',
+                        alignItems: 'center',
+                        padding: 12,
+                    }}
+                >
+                    <Button
+                        bsSize='large'
+                        bsStyle='large'
+                        title='Download'
+                        aria-label='Download'
+                        className='collabora-action-button'
+                        style={{
+                            border: 0,
+                            width: 40,
+                            height: 40,
+                            borderRadius: 4,
+                            marginBottom: 5,
+                            alignItems: 'center',
+                            display: 'inline-flex',
+                            justifyContent: 'center',
+                        }}
+                        href={Client.getFileUrl(fileInfo.id)}
+                        target='_blank'
+                        rel='noopener noreferrer'
+                        download={true}
+                    >
+                        <i className='fa fa-cloud-download'/>
+                    </Button>
+                    <div
+                        id='separator'
+                        style={{
+                            width: 1,
+                            height: 40,
+                            backgroundColor: '#e1e1e1',
+                            margin: '0 8px',
+                        }}
+                    />
+                    <CloseIcon
+                        id='closeIcon'
+                        title='Close'
+                        aria-label='Close'
+                        className='close-x'
+                        onClick={onClose}
+                        style={{
+                            position: 'relative',
+                            top: 'unset',
```

Comment:
There are too much inline styles in this file. Maybe we should create a separate css file.